_topic_maps/_topic_map.yml

```diff
@@ -3198,6 +3198,16 @@ Topics:
 - Name: Planning your environment according to object maximums
   File: planning-your-environment-according-to-object-maximums
   Distros: openshift-origin,openshift-enterprise
+- Name: Compute Resource Quotas
+  File: compute-resource-quotas
+  Topics:
+- Name: Admin Guide limits
+  Dir: quotas
+  File: limits
+- Name: Admin Guide Quota
+  Dir: quotas
+  File: quota
+  Distros: openshift-origin, openshift-enterprise
```

Comment:
```suggestion
- Name: Compute Resource Quotas
  File: compute-resource-quotas
- Name: Quotas
  Dir: quotas
  Topics:
  - Name: Admin Guide limits
    File: limits
  - Name: Admin Guide Quota
    File: quota
```

Reply:
OK, I think I've got the order and indents correct.

Reply:
@bscott-rh  I went ahead and converted to modules and condensed the file contents. Then I was able to delete the subdirectories as well as remove them from the topic map. I'm very happy at how much "cleaner" it is.